### ISSUE TITLE
Login button in mobile view doesn't appear

### ISSUE THREAD

**Issue by himanish-star** [himanish-star/Bumba-The-Assistant-App#8]
![image](https://user-images.githubusercontent.com/30361728/34826590-32932a00-f6fd-11e7-879c-c93397f48ff5.png)

**Comment by shikhar-scs** [comment]
Yup that issue does exist. Its related to popper.js I guess. For the time-being, mobile users can use the login button on the main carousel

**Comment by himanish-star** [comment]
Nope it exists. As it used to come before. Never mind I'll fix this now
